gfx.cpp

```diff
@@ -7773,68 +7764,73 @@ class GfxInternal
                     if(data != nullptr) populateRootConstants(program, program.parameters_, parameter, (uint32_t *)data);
                     cbv_desc.SizeInBytes = GFX_ALIGN(parameter.variable_size_, 256);
                 }
-                else if(parameter.parameter_->type_ == Program::Parameter::kType_Constants)
-                {
-                    cbv_desc.BufferLocation = allocateConstantMemory(parameter.parameter_->data_size_, data);
-                    if(data != nullptr) memcpy(data, parameter.parameter_->data_.constants_, parameter.parameter_->data_size_);
-                    cbv_desc.SizeInBytes = GFX_ALIGN(parameter.parameter_->data_size_, 256);
-                }
-                else if(parameter.parameter_->type_ == Program::Parameter::kType_Buffer)
+                else
```

Comment:
The diff from this point down looks ugly but all that happened was code was indented